tools/build_api.py

```diff
@@ -411,26 +411,25 @@ def get_mbed_official_release(version):
               RELEASE_VERSIONS
     """
 
+    # we ignore version for Mbed 6 as all targets in targets.json file are being supported
+    # if someone passes 2, we return emtpy tuple, if 5, we keep the behavior the same as 
+    # release version is deprecated and all targets are being supported that are present
+    # in targets.json file
+
+    if version == `2`:
+        raise InvalidReleaseTargetException("Mbed 2 not supported anymore with this version of tools")
+
     mbed_official_release = (
```

Comment:
I am not sure the syntax is correct here. Can you run the script with your changes?

Reply:
I was checking what CI expected and seems to be as it is here. will study CI results once we get some, if all is being tested as it should